async_geoprocess/src/cumulus_geoproc/processors/hrrr-total-precip.py

```diff
@@ -27,27 +27,32 @@
 
 
 @pyplugs.register
-def process(src: str, dst: str, acquirable: str = None):
-    """Grid processor
+def process(*, src: str, dst: str = None, acquirable: str = None):
```

Comment:
Why do we pass `*` as an argument? Is this needed?

Reply:
Any argument after `*` forces a keyword only.  Just a way to make sure that the argument is assigned correctly.